browser/brave_wallet/solana_provider_browsertest.cc

```diff
@@ -437,6 +437,8 @@ class SolanaProviderTest : public InProcessBrowserTest {
   SolanaProviderTest() {
     feature_list_.InitAndEnableFeature(
         brave_wallet::features::kBraveWalletSolanaFeature);
+    feature_list_.InitAndEnableFeature(
```

Comment:
minor: `InitWithFeatures({brave_wallet::features::kBraveWalletSolanaFeature, brave_wallet::features::kBraveWalletSolanaProviderFeature}, {});`

Reply:
Fixed in https://github.com/brave/brave-core/pull/13108/commits/9c8ecdeb97404d7a22009e9f80057835b089fbf0